### PR TITLE
Stitching: Add auto scaling on y direction in geomap kernel.

### PR DESCRIPTION
**Pull request by wujunkai166**

### CODE REVIEW

File: modules/ocl/cl_image_360_stitch.cpp [hunk 1]
```diff
@@ -545,7 +546,7 @@ CLImage360Stitch::update_scale_factors (uint32_t fm_idx, Rect crop_left, Rect cr
     float range = feature_center_x - center_x;
     XCAM_ASSERT (range > 1.0f);
     right_factor.x = (range + left_offsetx / 2.0f) / range;
-    right_factor.y = 1.0f;
+    right_factor.y = (_fisheye[left_idx].handler->get_stable_y_start () - left_offsety / 2.0f) / _fisheye[left_idx].handler->get_stable_y_start ();
     XCAM_ASSERT (right_factor.x > 0.0f && right_factor.x < 2.0f);
```
Comment (windyuan) [inline modules/ocl/cl_image_360_stitch.cpp]: is it possible get_stable_y_start () very near '0.0'?

File: modules/ocl/cl_image_360_stitch.cpp [hunk 2]
```diff
@@ -554,7 +555,7 @@ CLImage360Stitch::update_scale_factors (uint32_t fm_idx, Rect crop_left, Rect cr
     range = center_x - feature_center_x;
     XCAM_ASSERT (range > 1.0f);
     left_factor.x = (range + left_offsetx / 2.0f) / range;
-    left_factor.y = 1.0f;
+    left_factor.y = (_fisheye[left_idx].handler->get_stable_y_start () + left_offsety / 2.0f) / _fisheye[left_idx].handler->get_stable_y_start ();
     XCAM_ASSERT (left_factor.x > 0.0f && left_factor.x < 2.0f);
```
Comment (windyuan) [inline modules/ocl/cl_image_360_stitch.cpp]: is it possible get_stable_y_start () very near '0.0'?

**Comment by wujunkai166** [comment]
@windyuan

**Comment by windyuan** [comment]
@wujunkai166 please control the margin line to 120 characters.

**Comment by wujunkai166** [comment]
@windyuan

**Comment by windyuan** [comment]
merged. thanks.

**Pull request merged**